docker/generic/start_proxy.py

```diff
@@ -869,6 +869,15 @@ def make_argparser():
         in grpc-json transcoding. This is to support HTML 2.0<https://tools.ietf.org/html/rfc1866#section-8.2.1>.
         Set this flag to true to disable this feature.
         ''')
+    parser.add_argument(
+        '--exclude_colon_in_url_wildcard_path_segment', action='store_true',
```

Comment:
maybe just call "--exclude_colon_in_wildcard_path_match"

Reply:
How about we make the flag name more high level so it is in terms of the service config or end user behavior? Users don't know the details on path matching, colon, wildcards, etc.

Also it should be prefixed with transcoding, as users will only see behavior change in the transcoder with this flag.

`--transcoding_disallow_unregistered_custom_verbs`? Similar to https://github.com/grpc-ecosystem/grpc-httpjson-transcoding/pull/66

Reply:
@TAOXUY

Reply:
Prefer `--allow_colon_in_wildcard_path_match` as it actually unrelated to the gRPC transcoding but for the route match only.

Reply:
Hmm, I agree implementation wise it is a change in path matching.

But these routes are only used when gRPC transcoding is configured. gRPC pass through does not use custom verbs. OpenAPI has no concept of custom verbs. This is only used by google.api.http annotations, which is for transcoding.

So I would prefer to keep this flag as `--transcoding_...`. It will make it very clear to customers. Customers who don't use transcoding know that they don't need to worry about the flag

Reply:
We don't know if/how http backend users use the `:` in their path segment. For those users, they need to flip this flag for backward compatibility and `--transcoding_...` doesn't make sense to them.

For transcoding users, they should understand this flag as they defined the custom verb in their httpRule.